tests/drops_group_test.py

```diff
@@ -97,14 +98,17 @@
 sonic_drops_test               0                    0
 """
 
-dropstat_path = "/tmp/dropstat-27"
+
+def remove_tmp_dropstat_file():
+    # remove the tmp portstat
+    cache = UserCache("dropstat")
+    cache.remove_all()
 
 class TestDropCounters(object):
     @classmethod
     def setup_class(cls):
         print("SETUP")
-        if os.path.exists(dropstat_path):
-            shutil.rmtree(dropstat_path)
+        remove_tmp_dropstat_file()
```

Comment:
>remove_tmp_dropstat_file [](http://example.com/codeflow?start=8&length=24)

do you still need remove dir `dropstat_path`?

Reply:
I don't think so. The dropstat_path seems to refer to a non-relevant location.